modules/storages/spec/features/manage_project_storage_spec.rb

```diff
@@ -67,19 +63,14 @@
   let(:folder1_fileinfo_response) do
     {
       ocs: {
-        meta: {
-          status: 'ok'
-        },
         data: {
-          '11': {
-            status: 'OK',
-            statuscode: 200,
-            id: 11,
-            name: 'Folder1',
-            path: 'files/Folder1',
-            mtime: 1682509719,
-            ctime: 0
-          }
+          status: 'OK',
```

Comment:
Q: did the file info payload from nextcloud response change? 🤔  _Another great case for `VCR`, less ambiguity_ 😄

Reply:
Oh, I see it's now a `get` for a single folder